data/json/recipes/recipe_deconstruction.json

```diff
@@ -2201,7 +2201,7 @@
   "reversible": true,
   "autolearn": false,
   "qualities" : [ {"id":"SAW_M","level":1,"amount":1} ],
-  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ] ] ],
+  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ], [ "welder_crude", 750 ] ] ],
```

Comment:
I'm not able to use fully charged makeshift welder (450) with extra batteries in my inventory for crafting a drum using this receipt.